Gemfile

```diff
@@ -39,9 +39,8 @@ gem "govuk_message_queue_consumer", "~> 2.1.0"
 gem "govuk-lint", "~> 1.0.0"
 
 group :test do
-  gem "minitest", "~> 4.7.5"
+  gem 'test-unit-minitest', require: nil
```

Comment:
Double quotes and no `require`s please :) See https://github.com/alphagov/rummager/pull/620 for the why's.